config/overlays/dev/kustomization.yaml

```diff
@@ -1,28 +0,0 @@
-resources:
-- ../../crd/external
-- ../../default
-
```

Comment:
No. I've never used this overlay. It is not used to deploy with the operator.

I remember this is for developing. The only additional thing I see is applying CRDs, which are external. If that's the case, I don't really see the need of an overlay. We could manually apply the CRDs as part of dev setup.

Honestly, I don't fully understand the use-case, because testing is usually done by also deploying the other components and that would create the CRDs.